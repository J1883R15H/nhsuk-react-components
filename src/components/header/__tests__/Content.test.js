import React from 'react';
import { shallow } from 'enzyme';
import Content from '../components/content';

describe('Header.Content', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Content>Child</Content>);
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
