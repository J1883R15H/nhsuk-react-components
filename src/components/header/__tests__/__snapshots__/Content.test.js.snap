// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.Content matches snapshot 1`] = `
<div
  className="nhsuk-header__content"
>
  Child
</div>
`;
